x-pack/plugins/index_lifecycle_management/public/application/store/selectors/policies.js

```diff
@@ -193,8 +193,15 @@ const phaseFromES = (phase, phaseName, defaultEmptyPolicy) => {
     }
 
     if (actions.set_priority) {
-      policy[PHASE_INDEX_PRIORITY] = actions.set_priority.priority;
+      const { priority } = actions.set_priority;
+
+      if (priority) {
+        policy[PHASE_INDEX_PRIORITY] = priority;
```

Comment:
Nit: this is a good use case for `policy[PHASE_INDEX_PRIORITY] = priority ?? ''` and remove the `if else` statement.

Reply:
👍 Good point. Thanks for the reminder about this!